packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -207,6 +208,95 @@ const responseHeadersTest = async (req, res) => {
     res.json(jsonFromRequest(req))
 }
 
+/**
+ * Express handler that allocates a lot of memory, and then removes
+ * a reference to the objects, such that they may be garbage collected.
+ */
+// const memoryTest = async (req, res) => {
```

Comment:
@mitesh-patel-crm I would prefer uncommenting this code so that it becomes available on the playground environments. What do you think?